docs/blog/2019-07-03-customizing-styles-in-gatsby-themes-with-theme-ui/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Customizing Styles in Gatsby Themes with Theme UI"
+date: 2019-07-03
+author: Brent Jackson
+tags:
+  - themes
+  - css
+  - styles
+  - theme-ui
+---
+
+With Gatsby themes we want to make the experience of building a site as flexible and extensible as possible.
+Themes allow you to reuse site configurations, sets of plugins, and components across multiple Gatsby sites.
+We've tried to design the API of themes to [_progressively disclose complexity_][progressive disclosure],
+where smart defaults are wrapped into abstractions, but customization and configuration are always available at your fingertips.
+The styles used in our official themes are no exception.
+
+## Say hello to our official themes
+
+Today, we're releasing two official themes: a blog theme inspired by Dan Abromov's [overreacted.io][], and our notes theme,
+which is intended for freeform, non-linear content and inspired by Tom Critchlow's concept of a _[Digital Garden][]_.
+Both themes include their own styles so that you can build and publish a blog without starting from scratch.
+In order to make these themes truly _themeable_, the values used for typography, color, layout, and other design tokens
+can be customized by editing part or all of a `theme` configuration object.
+This also means you can create and publish your own theme based on the official one,
+with custom typography and colors, by editing a few plain objects instead of recreating entire components.
+
+For sites that have multiple themes installed,
+and to help ensure a consistent look and feel,
+you can choose which theme's styles take precedence based on their order in your Gatsby configuration.
+The styles provided by each theme are only applied to the pages it generates,
+giving you complete control of your site's styles.
+
+Huge thanks to our own Amberley Romo & John Otander for their incredible work on these themes.
+
+## Theme UI
+
+To enable this level of style customization,
+we've been working on a new open source library called _[Theme UI][]_.
+For authors of Gatsby themes, this library is entirely optional and in no way coupled to Gatsby themes,
+but we encourage you to check it out.
+Gatsby themes can be built without Theme UI, and Theme UI can be used without Gatsby.
```

Comment:
Love this level of detail! It really helps to answer these questions here. 🙌

Reply:
Thanks! I'm sort of anticipating some questions like this, and I don't want to scare anyone off 🙂